EventManagement/EventAPI/Domain/Event.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EventAPI.Domain
+{
+    public class Event
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public string EventImageUrl { get; set; }
+        public int MaxOccupancy { get; set; }
+        public int MaxTicketsPerUser { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public bool isPaidEvent { get; set; }
```

Comment:
Could you please change property names to PascalCase.
Add the missing CategoryId property to this class